Tests/UnitTests/Plant/ZadokPMFWheatTests.cs

```
@@ -0,0 +1,209 @@
+                $"Stage 55 DAS should be {expectedStage55}, but was {finalStage55}");
+            
+            Assert.That(finalStage65, Is.EqualTo(expectedStage65), 
+                $"Stage 65 DAS should be {expectedStage65}, but was {finalStage65}");
+            
+            Assert.That(finalStage90, Is.EqualTo(expectedStage90), 
+                $"Stage 90 DAS should be {expectedStage90}, but was {finalStage90}");
+        }
+
+        /// <summary>
+        /// Test that StageDAS returns 0 for stage index 1 before the stage is reached.
+        /// </summary>
+        [Test]
+        public void TestStageDAS_Stage1_BeforeReached()
+        {
+            // This test checks the behavior of StageDAS(1) to ensure it returns 0 before stage 1 is reached
+            string path = Path.Combine("%root%", "Tests", "Simulation", "Zadoks", "ZadoksStage.apsimx");
+            path = PathUtilities.GetAbsolutePath(path, null);
+            Simulations sims = FileFormat.ReadFromFile<Simulations>(path).Model as Simulations;
+            
+            foreach (Soil soil in sims.Node.FindChildren<Soil>(recurse: true))
+                soil.Sanitise();
+            
+            DataStore storage = sims.Node.FindChild<DataStore>(recurse: true);
+            storage.UseInMemoryDB = true;
+            Simulation sim = sims.Node.FindChild<Simulation>(recurse: true);
+
+            // Setup the report variables
+            Models.Report report = sim.Node.FindChild<Models.Report>(recurse: true);
+            report.VariableNames = [
+                "[Clock].Today",
+                "[Wheat].Phenology.Zadok.StageDAS(0)"
+            ];
+
+            sim.Prepare();
+            // The exception should be thrown when trying to run the simulation with an invalid stage index
+            Assert.Throws<SimulationException>(() => sim.Run(), 
+                "StageDAS(91) should throw an exception as it's out of valid range (1-90)");
```

Exception message should be `StageDAS(0) should throw an exception...`

---

Thanks for fixing. Forget to update message after pasting.